src/test/java/com/microsoft/sqlserver/jdbc/databasemetadata/DatabaseMetaDataForeignKeyTest.java

```diff
@@ -84,6 +84,34 @@ public static void setupVariation() throws SQLException {
                     + "(c41) ON DELETE set null ON UPDATE cascade," + " c15 int FOREIGN KEY REFERENCES "
                     + AbstractSQLGenerator.escapeIdentifier(table5) + "(c51) ON DELETE set default ON UPDATE no action,"
                     + ")");
+
+            TestUtils.dropSchemaIfExists(anotherSchema, stmt);
+            stmt.execute("CREATE SCHEMA " + AbstractSQLGenerator.escapeIdentifier(anotherSchema));
+
+            TestUtils.dropTableIfExists(AbstractSQLGenerator.escapeIdentifier(schema) + "."
```

Comment:
You just created the schema, there is no way it contains a table, these drops are unnecessary.